haas/templates/base.html

```diff
@@ -15,10 +15,25 @@
   <!--[if IE 8]>
     <link href="{{ url_for('static', filename='govuk-frontend-ie8-4.1.0.min.css') }}" rel="stylesheet">
   <![endif]-->
+
+  {{ super() }}
```

Comment:
This `{{ super() }}` isn't necessary any more, because `govuk_frontend_jinja/template.html` has an empty `head` block.